tests/composition/test_control.py

```diff
@@ -904,6 +904,45 @@ def test_multilevel_control(self, mode, benchmark):
         assert np.allclose(res, [40])
         benchmark(oComp.run, [5], bin_execute=mode)
 
+    @pytest.mark.control
+    @pytest.mark.composition
+    @pytest.mark.parametrize("mode", ["Python",
+                                      pytest.param("LLVM", marks=pytest.mark.llvm),
+                                      pytest.param("LLVMExec", marks=pytest.mark.llvm),
+                                      pytest.param("LLVMRun", marks=pytest.mark.llvm),
+                                     ])
+    def test_recurrent_control(self, mode, benchmark):
+        monitor = pnl.TransferMechanism(default_variable=[[0.0]],
+                                    size=1,
+                                    function=pnl.Linear(slope=1, intercept=0),
+                                    output_ports=[pnl.RESULT],
+                                    name='monitor')
+
+        rtm = pnl.RecurrentTransferMechanism(default_variable=[[0.0, 0.0]],
+                                            function=pnl.Logistic(gain=1.0),
+                                            matrix=[[1.0, -1.0],
+                                                    [-1.0, 1.0]],
+                                            integrator_mode=True,
+                                            integrator_function=pnl.AdaptiveIntegrator(rate=(1)),
+                                            initial_value=np.array([[0.0, 0.0]]),
+                                            output_ports=[pnl.RESULT],
+                                            name='rtm')
+    
+        controller = pnl.ControlMechanism(
+            monitor_for_control=monitor,
+            control_signals=[(pnl.NOISE, rtm)])
+        
+        comp = pnl.Composition()
+        comp.add_node(monitor)
+        comp.add_node(rtm)
+        comp.add_node(controller)
+        comp.run(inputs = {
+                    monitor: [[1], [5], [1], [5]],
+                    rtm: [[1,0], [1,0] ,[1,0], [1,0]]
+                },
+            bin_execute=mode
+        )
```

Comment:
This should check the result. it'll probably require changing the modulated parameter to target one of the functions.